### PR TITLE
Add avatar to user

### PR DESCRIPTION
**Pull request by juan-apa**
As a user I should be able to upload a picture to my profile
[Trello card](https://trello.com/c/MvKqvV9G)

### CODE REVIEW

File: app/controllers/api/v1/registrations_controller.rb [hunk 1]
```diff
@@ -3,16 +3,29 @@ module V1
     class RegistrationsController < DeviseTokenAuth::RegistrationsController
       protect_from_forgery with: :exception
       include Api::Concerns::ActAsApiRequest
+      skip_before_action :check_json_request, only: :update
+
+      def update
+        super()
```
Comment (mdornel) [inline app/controllers/api/v1/registrations_controller.rb]: no need `()` when are no params

File: spec/requests/users/update_user_spec.rb [hunk 2]
```diff
@@ -60,6 +60,26 @@
         end
       end
 
+      context 'with avatar field' do
+        let!(:header_param) { auth_headers(user).merge(accept: 'application/json') }
+        let!(:avatar_img) { fixture_file_upload('spec/fixtures/test_image.png', 'image/png') }
+        let(:user_id) { -1 }
```
Comment (mdornel) [inline spec/requests/users/update_user_spec.rb]: Why need this?

File: app/controllers/api/v1/registrations_controller.rb [hunk 3]
```diff
@@ -3,16 +3,29 @@ module V1
     class RegistrationsController < DeviseTokenAuth::RegistrationsController
       protect_from_forgery with: :exception
       include Api::Concerns::ActAsApiRequest
+      skip_before_action :check_json_request, only: :update
+
+      def update
```
[the remark this answers is not in the document]
Comment (mdornel) [reply app/controllers/api/v1/registrations_controller.rb]: Agree with @jdrosales17, also why you need to redefine it without any change? I mean, as it is now os doing exactly the same than the original.

Actually I think you can remove this, give a try.

[the remark this answers is not in the document]
Comment (juan-apa) [reply app/controllers/api/v1/registrations_controller.rb]: removing it doesn't break anything. I added the update function, thinking that the skip_before_action hook for update needed an implementation of the function in the class that's extending `DeviseToeknAuth`

**Pull request merged**